package.json

```diff
@@ -33,7 +33,7 @@
     "dockerode": "^2.4.3",
     "eslint": "^2.13.1",
     "eslint-config-loopback": "^4.0.0",
-    "loopback-datasource-juggler": "^3.0.0",
+    "loopback-datasource-juggler": "git://github.com/strongloop/loopback-datasource-juggler.git#couchdb/test",
```

Comment:
@b-admike yep yep, that juggler-branch will be checked in first, then here will be changed back.